include/igl/triangle/scaf.cpp

```diff
@@ -7,41 +7,41 @@
 // obtain one at http://mozilla.org/MPL/2.0/.
 
 #include "scaf.h"
+#include "triangulate.h"
 
 #include <Eigen/Dense>
 #include <Eigen/IterativeLinearSolvers>
 #include <Eigen/Sparse>
 #include <Eigen/SparseCholesky>
 #include <Eigen/SparseQR>
-#include <igl/PI.h>
-#include <igl/Timer.h>
-#include <igl/boundary_loop.h>
-#include <igl/cat.h>
-#include <igl/doublearea.h>
-#include <igl/flip_avoiding_line_search.h>
-#include <igl/flipped_triangles.h>
-#include <igl/grad.h>
-#include <igl/harmonic.h>
-#include <igl/local_basis.h>
-#include <igl/map_vertices_to_circle.h>
-#include <igl/polar_svd.h>
-#include <igl/slice.h>
-#include <igl/slice_into.h>
-#include <igl/slim.h>
-#include <igl/triangle/triangulate.h>
-#include "mapping_energy_with_jacobians.h"
-
-#include <iostream>
+#include "../PI.h"
+#include "../Timer.h"
```

Comment:
Hmm you don't have to change those include, `<igl/cat.h>` should still be valid.

Reply:
Yes, but I looked at other files and thought that this is part of the style.

Reply:
Do you think I should revert this?

Reply:
It probably doesn't matter much in this case, but using full path (`<igl/Timer.h>` is preferable for public headers, while relative paths should be used to point to private dependencies.